src/weekly/WeeklyExporter.js

```diff
@@ -147,3 +147,44 @@ export class CaHandDeliveryExporter extends CaExporter {
     return handDeliveryValue.trim().toUpperCase() === 'YES'
   }
 }
+
+const euCountries: string[] = [
```

Comment:
There are 4 other countries in the single market (to various degrees) Iceland, Liechtenstein, Norway and Switzerland. I think it'd be worth clarifying if they should also be included

Reply:
That's a good point.  I think it's unlikely but I'll check.

Reply:
I can confirm that this is only for the 27 EU member states.

Reply:
Excellent thanks